### PR TITLE
Added a check for the value of Omega_k.

### PR DESCRIPTION
**Pull request by t-ferreira**
Values of Omega_k < -1.0135 (approximately), causes the kernel to crash, so I added an error message.

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.6.0
```
Comment (elisachisari) [inline .python-version:1]: This looks like a user-specific file. Should it be unstaged?

Comment (t-ferreira) [reply .python-version:1]: My apologies! Yes, it should.

**Comment by damonge** [comment]
Just checking, is there anything particular about -1.0135?

**Comment by damonge** [comment]
If the value is approximate, it might be best to leave a small safety buffer and cut at -1 instead.

**Pull request merged**